flink-python/pyflink/fn_execution/operation_utils.py

```diff
@@ -310,7 +310,7 @@ def wrapped_keyed_process_function(value):
             on_timer_ctx.set_timestamp(value[1])
             on_timer_ctx.timer_service().set_current_watermark(value[2])
             current_key = value[3]
-            on_timer_ctx.set_current_key(current_key)
+            on_timer_ctx.set_current_key(current_key[0])
```

Comment:
What's the purpose of this change?

Reply:
The `current_key` is a Row that is wrapped in java operator before sending it to python process, therefore we need to unwrap it before setting to the `on_timer_ctx`.

Reply:
Is this a change specific for this PR or is it an issue which also exists in 1.12? If this is the latter case, we'd better split it into a separate commit or separate PR and also cherry pick to 1.12. What do you think?

Reply:
Actually, this issue also exists in 1.12. It would be better to create a new issue and fix it in a separate PR.